packages/i18n-calypso/src/i18n.js

```diff
@@ -165,9 +142,11 @@ function I18N() {
 		return new I18N();
 	}
 	this.defaultLocaleSlug = 'en';
+	// Tannin always needs a plural form definition, or it fails when dealing with plurals.
+	this.defaultPluralForms = n => ( n === 1 ? 0 : 1 );
 	this.state = {
 		numberFormatSettings: {},
-		jed: undefined,
+		tannin: undefined,
 		locale: undefined,
 		localeSlug: undefined,
 		textDirection: undefined,
```

Comment:
Noting: The assignment of `translations` state on the line following this one appears to be unused (also making the `'lru'` dependency unused in this file).